tests/media/video/test_video_import.py

```
@@ -0,0 +1,335 @@
+from unittest.mock import patch, MagicMock
+
+from django.test import TestCase
+
+from endoreg_db.services.video_import import VideoImportService
+from endoreg_db.models import Center, EndoscopyProcessor
+
+from ...helpers.default_objects import get_default_processor, get_default_center
+from ...helpers.data_loader import load_center_data, load_endoscope_data
+
+
+class TestVideoImportFileMovement(TestCase):
```

Pylint (4.0.1)

[refactor] Too many instance attributes (9/7)

(R0902)